tensorflow/contrib/layers/python/layers/feature_column.py

```diff
@@ -74,7 +74,7 @@
       columns=[department_column, bucketized_age_column],
       hash_bucket_size=1000)
 
-Example of building tf.learn model using FeatureColumns:
+Example of building `Estimator` model using FeatureColumns:
```

Comment:
It doesn't sound good grammatically. I'm thinking that this should be an Example of building *an* Estimator, for instance.